src/components/ExpenseForm.js

```diff
@@ -1,20 +1,26 @@
 import React, { Component } from "react";
-import { Button, Form, FormGroup, Label,  Input } from "reactstrap";
+import { Link, Redirect } from "react-router-dom";
+import { Button, Form, FormGroup, Label ,Input} from "reactstrap";
 
 import { Formik, Field, ErrorMessage } from "formik";
 import * as Yup from "yup";
 
-const AddExpenseSchema = Yup.object().shape({
+const LoginSchema = Yup.object().shape({
   title: Yup.string().required("Required title"),
   amount: Yup.string().required("Required Amount"),
   category: Yup.string().required("Required category")
 });
-class AddExpence extends Component {
+class ExpenseForm extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      isLoggedIn: false
+    };
   }
   render() {
+    if (this.state.isLoggedIn) {
+      return <Redirect to="/dashboard" />;
+    }
```

Comment:
I don't think I can understand why you want to redirect users who are signed in. Could you explain it to me, please?

Reply:
not needed. Will update.